community/inventory_service/inventory_service_tags_ukrainian.vdf

```diff
@@ -34,61 +34,61 @@
-		// where an item can be equipped.
-		"[english]inventory_slot"		"Slot"
-		"inventory_slot_medal"		"Medal"
-		"[english]inventory_slot_medal"		"Medal"
-		"inventory_slot_suit"		"Suit"
-		"[english]inventory_slot_suit"		"Suit"
-		"inventory_slot_weapon"		"Weapon"
-		"[english]inventory_slot_weapon"		"Weapon"
-		"inventory_slot_equipment"		"Equipment"
-		"[english]inventory_slot_equipment"		"Equipment"
-		"inventory_class_restriction"		"Class Restriction"
-		// for equippable items that can only be used on one class.
-		"[english]inventory_class_restriction"		"Class Restriction"
-		"inventory_class_restriction_officer"		"Officer"
-		"[english]inventory_class_restriction_officer"		"Officer"
-		"inventory_class_restriction_specialweapons"		"Special Weapons"
-		"[english]inventory_class_restriction_specialweapons"		"Special Weapons"
-		"inventory_class_restriction_medic"		"Medic"
-		"[english]inventory_class_restriction_medic"		"Medic"
-		"inventory_class_restriction_tech"		"Tech"
-		"[english]inventory_class_restriction_tech"		"Tech"
-		"inventory_crafting_item"		"Crafting Item"
-		"[english]inventory_crafting_item"		"Crafting Item"
-		"inventory_crafting_item_single_use_tool"		"Single-Use Tool"
-		"[english]inventory_crafting_item_single_use_tool"		"Single-Use Tool"
-		"inventory_crafting_item_ultra_common"		"Ultra-Common"
-		// common crafting materials. (100-1000 drop at a time)
-		"[english]inventory_crafting_item_ultra_common"		"Ultra-Common"
-		"inventory_crafting_item_common_part"		"Common"
-		// common crafting materials. (1-3 drop at a time)
-		"[english]inventory_crafting_item_common_part"		"Common"
-		"inventory_crafting_item_alien_part"		"Alien Part"
-		// uncommon crafting materials.
-		"[english]inventory_crafting_item_alien_part"		"Alien Part"
-		"inventory_crafting_item_computer_part"		"Computer Part"
-		// rare crafting materials.
-		"[english]inventory_crafting_item_computer_part"		"Computer Part"
-		"inventory_crafting_item_regional"		"Regional"
-		// crafting items that can be found by playing some campaigns but not others.
-		"[english]inventory_crafting_item_regional"		"Regional"
-		"inventory_strange"		"Strange Devices"
-		// displayed before the list of Strange item accessories, eg. Strange Devices: Missions, Aliens Killed
-		"[english]inventory_strange"		"Strange Devices"
-		"item_store_category_donations"		"Donations"
-		// items that are essentially receipts for donating money to someone in the community. (for the forseeable future, all items on the store will be in this category)
-		"[english]item_store_category_donations"		"Donations"
-		"item_store_category_mapstamp"		"Map Stamps"
-		// "postage stamps" that cost $1 (or equivalent), with the proceeds going to a mapper.
-		"[english]item_store_category_mapstamp"		"Map Stamps"
+"inventory_rarity_genuine"		"Genuine"
+// real-money donation receipt items from the store.
+"[english]inventory_rarity_genuine"		"Genuine"
+"inventory_rarity_strange_rarity"		"Strange"
```

Comment:
Were these two intentionally kept the same as English?

Reply:
No, I must have missed those lines by accident.